src/pages/TeamPage/TeamPage.tsx

```diff
@@ -149,8 +162,16 @@ export class TeamPage extends React.Component<ITeamPageProps, ITeamPageState> {
           </FlexCell>
 
           <FlexCell>
-            <Mo2PercentWidget
-              mo2={this.state.mo2}
+            <EnduranceZoneWidget
+              enduranceZone={this.state.enduranceZone}
+              heightPx={300}
+              widthPx={300}
```

Comment:
With some of jonas’ changes we are now using 900 for the width and putting them on their own line

Reply:
I would merge master into your branch, the changes to width and layout are there

Reply:
Yep merged and fixed!